cmd/chaos-multiversion-helper/create/create.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2022 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package create
+
+import (
+	"go/ast"
+	"go/parser"
+	"go/printer"
+	"go/token"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func NewCreateCmd(log logr.Logger) *cobra.Command {
+	var from, to string
+	var asStorageVersion bool
+
+	var cmd = &cobra.Command{
+		Use:   "create --from <old-version> --to <new-version>",
+		Short: "Create a new version of chaos api",
+		Run: func(cmd *cobra.Command, args []string) {
+			err := run(log, from, to, asStorageVersion)
+			if err != nil {
+				log.Error(err, "create new version")
+				os.Exit(1)
+			}
+		},
+	}
+
+	cmd.Flags().StringVar(&from, "from", "", "old version of chaos api")
+	cmd.Flags().StringVar(&to, "to", "", "new version of chaos api")
+	cmd.Flags().BoolVar(&asStorageVersion, "as-storage-version", true, "new version of chaos api")
+
+	cmd.MarkFlagRequired("from")
+	cmd.MarkFlagRequired("to")
+
+	return cmd
+}
+
+func run(log logr.Logger, from, to string, asStorageVersion bool) error {
+	fileSet := token.NewFileSet()
+
+	oldAPIDirectory := "api/" + from
+	newAPIDirectory := "api/" + to
+	err := os.Mkdir(newAPIDirectory, 0755)
```

Comment:
Hi @YangKeao, I got error here when I ran the command for the second time (it worked fine for the first time), because the directory is already exist.

Maybe we will meet a situation that run `make migrate-version` failed in some steps, and the directory or file has been created, and when we try the command again, it just 💥 .

Reply:
I think it's by design. The user should run these commands in a pure new revision, and if they meet any error, the best way to solve is `git reset --hard` :crying_cat_face:

Reply:
(Though, `git reset --hard` seems not to work for newly created file, maybe `git add * && git reset --hard` is better.)

Reply:
> I think it's by design. The user should run these commands in a pure new revision, and if they meet any error, the best way to solve is git reset --hard 😿

SGTM, but maybe we could add this tip to `README.md` or any other docs?

Reply:
> SGTM, but maybe we could add this tip to README.md or any other docs?

Done.